frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -172,56 +173,59 @@ export default memo(function Filters(): JSX.Element {
 
   return (
     <TooltipWrapper>
-      <div>
-        <ComplexFilter
-          multiple
-          label="Dataset"
-          options={datasets as unknown as DefaultMenuSelectOption[]}
-          onChange={handleDatasetsChange}
-          value={selectedDatasets as unknown as DefaultMenuSelectOption[]}
-          InputDropdownComponent={
-            StyledComplexFilterInputDropdown as typeof ComplexFilterInputDropdown
-          }
-          MenuSelectProps={MenuSelectProps}
-          InputDropdownProps={InputDropdownProps}
-        />
-        <ComplexFilter
-          multiple
-          label="Disease"
-          options={disease_terms}
-          onChange={handleDiseasesChange}
-          value={selectedDiseases}
-          InputDropdownComponent={
-            StyledComplexFilterInputDropdown as typeof ComplexFilterInputDropdown
-          }
-          MenuSelectProps={MenuSelectProps}
-          InputDropdownProps={InputDropdownProps}
-        />
-        <ComplexFilter
-          multiple
-          label="Ethnicity"
-          options={ethnicity_terms}
-          onChange={handleEthnicitiesChange}
-          value={selectedEthnicities}
-          InputDropdownComponent={
-            StyledComplexFilterInputDropdown as typeof ComplexFilterInputDropdown
-          }
-          MenuSelectProps={MenuSelectProps}
-          InputDropdownProps={InputDropdownProps}
-        />
-        <ComplexFilter
-          multiple
-          label="Sex"
-          options={sex_terms}
-          onChange={handleSexesChange}
-          value={selectedSexes}
-          InputDropdownComponent={
-            StyledComplexFilterInputDropdown as typeof ComplexFilterInputDropdown
-          }
-          MenuSelectProps={MenuSelectProps}
-          InputDropdownProps={InputDropdownProps}
-        />
-      </div>
+      <Wrapper>
```

Comment:
adding `<Wrapper />` around secondary filter section and the sort section, so they flex by column